### ISSUE TITLE
Automatically unwrap MPtr Julia "objects" ?

### ISSUE THREAD

**Issue by fingolfin** [oscar-system/GAPJulia#27]
I wrote a Julia function which just returns its first argument, and put it into GAP variable `myfunc`. Now I get this:
```
gap> myfunc(fail, 2);
<Julia: Main.ForeignGAP.MPtr()>
```
It would be very convenient (and should be super cheap!) to instead return a this "MPtr" object as a regular GAP object. I.e., so that I'd get:
```
gap> myfunc(fail, 2);
fail
```

**Comment by sebasguts** [comment]
Indeed, I will do this. Now with the integrated gc, this is sensible.
